deploy/cli/dapps.py

```diff
@@ -195,8 +196,17 @@ def get_solana_accounts_in_tx(eth_transaction):
         )
 
 
+def get_transaction_by_hash(eth_transaction):
```

Comment:
use NeonWeb3Client here instead if raw request. You can create client object in this file in one place and reuse it